test/Npgsql.Tests/Types/EnumTests.cs

```diff
@@ -170,11 +170,11 @@ public async Task Unmapped_enum_as_clr_enum_supported_only_with_EnableUnmappedTy
             nameof(NpgsqlDataSourceBuilder));
 
         var exception = await AssertTypeUnsupportedWrite(Mood.Happy, enumType);
-        Assert.IsInstanceOf<NotSupportedException>(exception.InnerException);
+        Assert.That(exception.InnerException, Is.InstanceOf<NotSupportedException>());
```

Comment:
As with IsNull, previous seems slightly better to me...

Reply:
Idem